addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -143,6 +154,8 @@ const FormEditor = options.Class.extend({
             $(template.content.querySelector('.s_website_form_field_description')).replaceWith(field.description);
         }
         template.content.querySelectorAll('input.datetimepicker-input').forEach(el => el.value = field.propertyValue);
+        template.content.querySelectorAll('[name]').forEach(el => el.name = this._getNameWithoutQuotes(el.name));
+        template.content.querySelectorAll('[data-name]').forEach(el => el.dataset.name = this._getNameWithoutQuotes(el.dataset.name));
```

Comment:
I don't understand why we need this and what it does, why is the upfront encoding when the value is set not enough (the other part of the diff)?
It's probably related to your previous message on the PR but I can't wrap my head around it.
If you could explain again :)